app/views/enrich.njk

```diff
@@ -33,25 +33,23 @@
             <thead class="govuk-table__head">
               <tr class="govuk-table__row">
                 <th scope="col" class="govuk-table__header govuk-body-s">Scheme</th>
+                <th scope="col" class="govuk-table__header govuk-body-s">Scheme Year</th>
                 <th scope="col" class="govuk-table__header govuk-body-s">FRN</th>
-                <th scope="col" class="govuk-table__header govuk-body-s">Application ID</th>
-                <th scope="col" class="govuk-table__header govuk-body-s">Invoice</th>
-                <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-body-s">Request</th>
+                <th scope="col" class="govuk-table__header govuk-body-s">Agreement number</th>
                 <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-body-s">Total amount</th>
-                <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-body-s">Arrived</th>
+                <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-body-s">Days waiting</th>
                 <th scope="col" class="govuk-table__header"></th>
               </tr>
             </thead>
             <tbody class="govuk-table__body">
               {% for data in enrichData %}
                 <tr class="govuk-table__row">
                   <td scope="col" class="govuk-table__cell govuk-body-s">{{ data.schemes.name }}</td>
+                  <td scope="col" class="govuk-table__cell govuk-body-s">{{ data.marketingYear }}</td>
```

Comment:
Should marketing year be tagged as `govuk-table__cell--numeric`, also <-- what does that do?

Reply:
i think it right aligns the cell data for easy comparision between columns according to the docs [here](https://design-system.service.gov.uk/components/table/#numbers-in-a-table)

Therefore dont think we need it as right align makes it look odd.